### PR TITLE
Add var command

### PR DESCRIPTION
**Pull request by DanMayhem**
I understand that the way forward with environment variables is through custom hooks in each environment. However, as I began working on the hooks implementation for pull request #47, it felt less and less well suited to handling environment variables. 

So instead I implemented a `pew var` command that will set, display and unset enviornment variables in an environment. The variables are stored in an `.env` file in the virtual environment. This file takes the same format as forman `.env` files. This file is then read in `inve()` and the variables are added to the environment prior to the `check_call()` call.

I don't exepct that this will be merged, but I wanted to call it out as an alternative to implementing a hook in every environment that requires environment variables.

### CODE REVIEW

File: pew/pew.py [hunk 1]
```diff
@@ -81,6 +82,42 @@ def get_project_dir(env):
 
     return project_dir
 
+#class to support reading ini file
```
Comment (ryanhiebert) [inline pew/pew.py]: This might be better suited and expanded as a docstring for the `EnvParser` class.

**Comment by ryanhiebert** [comment]
I like the approach. Of course, I should, since I think I'm the one that suggested using a Forman-style `.env` file. For my case in particular, entering the project directory and loading the environment variables are the main things that I need to do when I enter a virtual environment, so it cleanly supports my needs.

**Comment by berdario** [comment]
A few days ago I closed #47, with the rationale of not adding complexity

The purpose of this PR was a simpler alternative to the hook system... but as I mentioned in the other PR, this functionality is already available elsewhere.

I'd suggest either [direnv](https://direnv.net/), or [pipenv](https://docs.pipenv.org/advanced.html#automatic-loading-of-env)'s builtin functionality

Sorry for not engaging before :/

Anyhow, I don't really like being too eager with closing tickets... so I'd rather wait for some feedback
I don't use .env or .envrc files myself, but maybe someone who does found some shortcoming in other tools and can make a case for having this functionality builtin?

**Comment by DanMayhem** [comment]
I've switched over to pipenv, and it is serving all my needs.

**Pull request closed without merging**